cdap-master/src/main/java/co/cask/cdap/data/tools/UpgradeTool.java

```diff
@@ -96,8 +96,8 @@
    * Set of Action available in this tool.
    */
   private enum Action {
-    UPGRADE("Upgrades CDAP to 3.2\n" +
-              "  The upgrade tool upgrades the following: \n" +
+    UPGRADE("Upgrades CDAP to " + ProjectInfo.getVersion().toString() +
```

Comment:
Sorry, disregard my comment. I was looking at this Version class as reference:
https://github.com/caskdata/cdap/blob/develop/cdap-proto/src/main/java/co/cask/cdap/proto/Version.java

The Version class you're using here has a toString implemented.